src/client/src/components/form/form.js

```diff
@@ -2,8 +2,21 @@ import { TextField } from "@mui/material";
 import { styled } from "@mui/system";
 import { theme } from "../../AppTheme.js";
 
-export const getInputFieldBackgroundColor = errorFieldName =>
-  errorFieldName ? theme.palette.error.background : "transparent";
+export const getFormFieldBackgroundColor = (fieldName, errors) => {
+  if (typeof fieldName === "string") {
```

Comment:
Q: Wozu brauchts den typecheck hier? In welchem Fall kann fieldName einen andere Typ haben?
Sonst PP: Early return
if (typeof fieldName !== "string") {
  return "transparent";
}
var fieldNameElements... 

PP: const statt var